README

```diff
@@ -195,3 +195,24 @@ Further Information
 -------------------
 http://igs.bkg.bund.de/index_ntrip.htm
 euref-ip@bkg.bund.de
+
+
+Matternet Addendum
+------------------
+We have now forked this open source software in our github.
+
+Expected usage:
+
+./ntripclient -s <server> -m <mount point> -r <port> -u <Matternet username>
+  -p <Matternet password> -n <initial NMEA GGA sentence>
```

Comment:
Could we add which parameters are required, if any?

Reply:
Done!